Sources/SwiftInspectorAnalyzers/Tests/PropertyAnalzyerSpec.swift

```diff
@@ -28,6 +28,7 @@ import Foundation
 import SwiftInspectorTestHelpers
 
 @testable import SwiftInspectorAnalyzers
+@testable import SwiftInspectorVisitors
```

Comment:
curious if we really need a testable import here

Reply:
We eventually delete this file 👌

Reply:
And in the new file we use the `@testable` to test the internal `merge(...)` method.